src/utils.ts

```diff
@@ -81,3 +81,30 @@ export function getObsidianURI(vault: Vault, path: string): string {
   const p = encodeURIComponent(path);
   return `obsidian://open?vault=${v}&file=${p}`;
 }
+
+export function logLevelFilter(logEvent: LogEvent, logLevel: LogLevel) {
+  if (logLevel === "debug") {
```

Comment:
I'd rather see a `switch` statement here to make it a bit cleaner. Also, I propose using `Array.includes()` instead of the `&&` combos, e.g.

```javascript
switch (logLevel) {
	case "debug":
    return true

  case "info":
    return logEvent.level !== "debug"

	case "warn":
	  return !["debug", "info", "log"].includes(logEvent.level)

	case "error":
	  return !["debug", "info", "log", "warn"].includes(logEvent.level)
}
```